aspen/dispatcher.py

```diff
@@ -120,7 +121,7 @@ def get_wildleaf_fallback():
         wild_nonleaf_ns = [ n for n in maybe_wild_nodes if not is_leaf_node(n) ]
 
         # store all the fallback possibilities
-        remaining = reduce(traverse, nodepath[depth:])
```

Comment:
It's passed in as a param, and is used correctly ( to join fs paths ) in other placrs. This spot was actually rejoining URL path parts.